run.sh

```diff
@@ -155,6 +156,9 @@ case "$deployment_action" in
     "gather_logs")
         gather_logs
         ;;
+    "test")
+        deploy_tempest
```

Comment:
This function was only added to deployment-actions-kvm.sh, we also want one in deployment-actions-openstack.sh